### PR TITLE
Feature/sg 493 modelnames instead of strings

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py [hunk 1]
```diff
@@ -23,35 +34,27 @@
 # THIS IS WHERE THE MAGIC HAPPENS- SINCE TRAINER'S CLASSES ATTRIBUTE WAS SET TO BE DIFFERENT FROM CITYSCAPES'S, AFTER
 # LOADING THE PRETRAINED REGSET, IT WILL CALL IT'S REPLACE_HEAD METHOD AND CHANGE IT'S SEGMENTATION HEAD LAYER ACCORDING
 # TO OUR BINARY SEGMENTATION DATASET
-model = models.get("regseg48", pretrained_weights="cityscapes", num_classes=1)
+model = models.get(Models.RREGSEG48, pretrained_weights="cityscapes", num_classes=1)
```
Comment (ofrimasad) [inline src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py:37]: too many R in REGSEG....

**Review by ofrimasad — COMMENTED** [review]
one minor comment

File: src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py]: fixed in latest commit.

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**